tests/download-util.test.ts

```diff
@@ -51,11 +45,7 @@ describe('#downloadAndExtractTool', function () {
   });
 
   it('downloads and extracts linux version', async function () {
-    if (os.platform() === 'win32') {
```

Comment:
This issue is from 2019, I'm curious if it's still an issue...

Reply:
I tried to remove this but it still errored. I wasn't able to investigate.

Reply:
Yea - me too. I'm going to add it back in for now to separate debugging from this small refactor.